updateCurrentIteration.js

```diff
@@ -7,6 +7,7 @@ var STATUS_COLUMN = 'B';
 var START_OF_ITERATION_STRING = 'current';
 var END_OF_ITERATION_STRING = 'next';
 var ACCESS_TOKEN = ''; // Required, ask someone.
+var RATE_LIMIT = 1000;
```

Comment:
If the only reason for this is to slow the script down so it doesn't hit the limit. Then I think we can speed it up as much as we can without hitting the limit. For example, request_limit / minute.
```
60000÷99=606
```
At this rate the requests should never be more then 99 in a min.

Reply:
Sounds good, will update it... I was to lazy to do that simple maths, figured a second was close enough!